### PR TITLE
Fix foonathan issues

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
See https://github.com/RoboStack/ros-humble/issues/32

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
Hi @traversaro - I'm blindly coding, could you please have a look?

File: vinca/templates/activate.sh.in [hunk 1]
```diff
@@ -22,5 +22,9 @@ esac
 export ROS_ETC_DIR=$CONDA_PREFIX/etc/ros
 export AMENT_PREFIX_PATH=$CONDA_PREFIX
 
+if [ "$OSTYPE" = "darwin*" ]; then
+    export RMW_IMPLEMENTATION="rmw_cyclonedds_cpp"
+fi
```
Comment (traversaro) [inline vinca/templates/activate.sh.in]:
```suggestion
if [[ "$OSTYPE" == darwin* ]]; then
    export RMW_IMPLEMENTATION="rmw_cyclonedds_cpp"
fi
```

On bash, this works (See https://stackoverflow.com/questions/2172352/in-bash-how-can-i-check-if-a-string-begins-with-some-value). not sure on zsh

**Comment by Tobias-Fischer** [comment]
Hmmm .. what if we use the case statement above that we know works?

**Comment by traversaro** [comment]
> Hmmm .. what if we use the case statement above that we know works?

Good idea!

**Pull request merged**